src/language/tekster_nn.ts

```diff
@@ -516,6 +516,9 @@ export default {
 
   'arbeidssituasjon.spm': 'What is your work situation?',
   'arbeidssituasjon.spm.hjelpetekst': 'You can select more than one answer',
+  'arbeidssituasjon.spm.hjelpetekst-åpne': 'The reason we ask about this',
+  'arbeidssituasjon.spm.hjelpetekst-innhold':
+    'We need information about your work situation in order to assess whether you are entitled to receive transitional benefits for single parents. As a general rule you are requried to be at least 50 per cent engaged in some form of occupational activity. In some cases we make exceptions to this rule.<br/><br/>Read more about these requirements <a href="https://www.nav.no/overgangsstonad-enslig#aktivitet">here</a>.',
```

Comment:
Følte det var litt brå overgang før siste setning. Kunne man hatt typ `However, there are some exceptions to this rule` or `However, in some cases...` for å få de til å henge mer sammen?

We need information about your work situation in order to assess whether you are qualified for transitional benefit. for single parents. You must be in at least 50 per cent occupational activity to qualify. However, there are some exceptions to the general rule.

Reply:
Ja, det var bedre. Går for en blanding av de to:

We need information about your work situation in order to assess whether you are qualified for transitional benefit for single parents. You must be at least 50 per cent engaged in some form of occupational activity to qualify. However, there are some exceptions to the general rule.